packages/cyberstorm-styles/src/index.tsx

```diff
@@ -0,0 +1,3 @@
+import "./styles/globals.css";
```

Comment:
The original tickets (TS-1214 & TS-1215) mention that user should be able to import globals and reset separately. Is there a feasible way to achieve this?